DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -37,131 +36,133 @@
+                  </svg>
+                </a>
+              </li>
+            }
+            @if (User.Identity.IsAuthenticated) {
+              <li class="nhsuk-header__navigation-item">
+                <a class="nhsuk-header__navigation-link" href="@Configuration["AppRootPath"]/MyAccount">
+                  My account
+                  <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                    <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                  </svg>
+                </a>
+              </li>
+              @if (User.GetAdminId() != null) {
                 <li class="nhsuk-header__navigation-item">
-                  <a class="nhsuk-header__navigation-link" asp-controller="Register" asp-action="Index">
-                    Register
+                  <a class="nhsuk-header__navigation-link" asp-controller="ApplicationSelector" asp-action="Index">
+                    Switch application
                     <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
                       <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
                     </svg>
                   </a>
                 </li>
               }
-              @if (User.Identity.IsAuthenticated) {
-                <li class="nhsuk-header__navigation-item">
-                  <a class="nhsuk-header__navigation-link" href="@Configuration["AppRootPath"]/MyAccount">
-                    My account
-                    <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
-                      <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
-                    </svg>
-                  </a>
-                </li>
-                @if (User.GetAdminId() != null) {
-                  <li class="nhsuk-header__navigation-item">
-                    <a class="nhsuk-header__navigation-link" asp-controller="ApplicationSelector" asp-action="Index">
-                      Switch application
-                      <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
-                        <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
-                      </svg>
-                    </a>
-                  </li>
-                }
+              var application = (string)ViewData["Application"] ?? "";
+              if (application != "Tracking System") {

```

Comment:
Could we pull this at the top of the page and have:
```
var shouldDisplayLogOutTab = application != "Tracking System"
```

We are planning on removing the log out tab on some other sub-applications so the logic will get a bit longer in the future. 🙏